lib/cosmos/system/target.rb

```diff
@@ -124,7 +123,14 @@ def process_file(filename)
           usage = "#{keyword} <FILENAME>"
           parser.verify_num_parameters(1, 1, usage)
           begin
-            Cosmos.require_file(parameters[0])
+            # Determine if this file is in our target lib and should be namespaced
+            filename = File.join(@dir, 'lib', parameters[0])
+            if File.exist? filename
+              const = find_or_create_const(@name)
+              const.module_eval(File.read(File.join(@dir, 'lib', parameters[0])))
```

Comment:
This works file for my tests even when I require another file. Not sure if / when this would break down.